ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -309,6 +309,7 @@ module Configuration = struct
 
   let watch_other_configs ~__context delay =
     let loop (token, was_in_rpu) =
+      let __context = Xapi_slave_db.update_context_db __context in
```

Comment:
As discussed in person, this does not look to be enough to get the event loop use the local DB, because the `Event.from` call still goes through the API, and the master.